code-challenges.test.js

```diff
@@ -1,51 +1,158 @@
-// Expected output: [ 2, 0, -1, 0 ]
 const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
-
-
-// b) Create the function that makes the test pass.
-
-
-// --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
-
-// a) Create a test with an expect statement using the variables provided.
-
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
-
 
-// b) Create the function that makes the test pass.
+const arrayRemainder = array => {
+  let numbersOnly = array.filter(Number.isFinite)
+  return numbersOnly.map(value => value % 3)
+}
+//  console.log(arrayRemainder(hodgepodge1)) 
+// console.log(arrayRemainder(hodgepodge2))
+
+const newRemainder = (array) => {
+  return array.filter(value => typeof value ==='number').map(value => value % 3)
+}
+
+console.log(newRemainder(hodgepodge1))
+console.log(newRemainder(hodgepodge2))
+
+// PASS  week-6-assessment-ecostrong/code-challenges.test.js
+// ● Console
+
+//   console.log
+//     [ 2, 0, -1, 0 ]
+
+//     at Object.log (week-6-assessment-ecostrong/code-challenges.test.js:97:10)
+
+//   console.log
+//     [ 2, 1, -1 ]
+
+// // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
+// // a) Create a test with an expect statement using the variables provided.
+describe("arrayCubed", () => { 
+ it("Create a function that takes in an array of numbers and returns the sum of all the numbers cubed", () => {
+      const cubeAndSum1 = [2, 3, 4]
+// // // // Expected output: 99
+       const cubeAndSum2 = [0, 5, 10]
+// // // // Expected output: 1125
+       expect(arrayCubed(cubedSum1)).toEqual("2, 0, -1, 0 ")
+       expect(arrayCubed(cubedSum2)).toEqual(" 2, 1, -1")
+      })
+})
+// FAIL  week-6-assessment-ecostrong/code-challenges.test.js
+// ● arrayCubed › Create a function that takes in an array of numbers and returns the sum of all the numbers cubed
+//  ReferenceError: arrayCubed is not defined
+// // b) Create the function that makes the test pass.
+
+// Psuedo Code
+// create a function that takes in the inputs of an array and returns the sum of all numbers cubed.
+//Use the reduce() method to find or calculate the sum of an array of numbers.
+//The reduce() method executes the specified reducer function on each member of the array resulting in a single output value, 
+
+const sum = array.reduce(function(a,b) {
+      return a + b;
+}, 0);
+
+console.log(sum)   
+const numbersCubed = array.map(arrayItem => typeof arrayItem * 3).sum  
+//Didn't work 
+
+const numCubed =(numbers) => {
+  let sum = 0; 
+  //iterate through numbers 
+  numbers.forEach((num) => {
+    //add to sum the number cubed 
+    sum+= num * num * num;
+  });
+  //return cubed sum
+  return sum;
+}
+//Can't run test error on line 159  Jest encountered an unexpected token
```

Comment:
Your function in your test and function name here do not match and will likely throw an error.